crates/esthri-cli/src/cli_opts.rs

```diff
@@ -228,3 +228,13 @@ pub struct EsthriServeParams {
     #[clap(long)]
     pub allowed_prefixes: Vec<String>,
 }
+
+#[derive(Args, Debug, Clone)]
+pub struct EsthriPresignParams {
+    /// The bucket to target (example: my-bucket)
+    #[clap(long)]
+    pub bucket: String,
+    /// The key to target (example: a/key/name.bin)
+    #[clap(long)]
+    pub key: String,
```

Comment:
Do we want to add a "method" parameter here?  Will differ from `aws s3` but I think could be useful.

Reply:
I'll do this

Reply:
we can work on this later